packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,4 +1,21 @@
 const request = require('../exporters/common/request')
+
+function getHeaders (reqType, debug, application) {
+  const headers = {
+    'Content-Type': 'application/json',
```

Comment:
You have changed some keys chars from lowercase to uppercase, `Content-Type`, `DD-Telemetry-API-Version`...
Is this ok?

Reply:
Yes, the specification states that the names should be in camelcase and some system-tests check it

Reply:
Hmm, but CamelCase is not like: `ContentType`, `DDClientLibraryLanguage` without `-`?

Reply:
CamelCaseHyphenSeparated :joy:

Reply:
Headers are case-insensitive, I don't understand why we're checking the casing anywhere.

Reply:
Node.js also automatically lower-cases headers in a bunch of places.

Reply:
ok, system-tests changed to be case-insensitive https://github.com/DataDog/system-tests/pull/1111/commits/5230d8f350ec21fa63ea8b57785990e905d8dfa3

Reply:
and restored [original header capitalization ](https://github.com/DataDog/dd-trace-js/pull/3055/files#diff-d42dccc371917e1e5b6b4961ffd59ac8e264c4c6a8c650b940f388ebbdf009c3R5)